milv.config.yaml

```diff
@@ -0,0 +1,6 @@
+white-list-external: ["localhost", "console.kyma.local"]
```

Comment:
Where are the other items in the previous whitelist?

Reply:
Right now they are not needed. List should be updated as needed and as short as possible.